templates/CRM/EasyBatch/Form/FinancialBatch.tpl

```diff
@@ -13,10 +13,13 @@ CRM.$(function($) {
   })
   var batchLabelHtml = $('#batchLabel').html();
   var batchContentHtml = $('#batchContent').html();
+
   // Contribution.
-  $(".crm-contribution-form-block-receive_date, .crm-payment-form-block-trxn_date, .crm-membership-form-block-receive_date, .crm-membershiprenew-form-block-receive_date")
-    .after($('tr.crm-contribution-form-block-batch_id'));
+  $("#receiptDate, .crm-payment-form-block-trxn_date, .crm-membership-form-block-receive_date, .crm-membershiprenew-form-block-receive_date").after($('tr.crm-contribution-form-block-batch_id'));
```

Comment:
@JoeMurray changed this line. Here's how it look now:
### CC Contribution Form
<img width="517" alt="screen shot 2019-01-08 at 11 48 13 pm" src="https://user-images.githubusercontent.com/3735621/50850550-2ea87d80-13a0-11e9-9867-ed78601fc93b.png">

### Cash,check Contribution Form
<img width="531" alt="screen shot 2019-01-08 at 11 47 46 pm" src="https://user-images.githubusercontent.com/3735621/50850584-497af200-13a0-11e9-9f31-32cf3cb829ef.png">